EIPS/eip-5516.md

```diff
@@ -0,0 +1,168 @@
+---
+eip: 5516
+title: Soulbound, Multi Token Standard.
+description: An interface for non-transferrable NFTs binding to an Ethereum account for standardized blockchain-based academic certification.
+author: Lucas Martín Grasso Ramos(@LucasGrasso), Matias Arazi(@MatiArazi)
+discussions-to: https://ethereum-magicians.org/t/eip-5516-soulbound-multi-token-standard/10485
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-19
+requires: 165, 1155
+---
+
+## Abstract
+
+This is a standard interface for non fungible, double signature, multi-token, Soulbound Token standard. It was inspired by the main characteristics of the EIP-1155 token and by the [article](https://vitalik.ca/general/2022/01/26/soulbound.html) and [paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4105763) published by Vitalik Butherin, in which benefits and potential use-cases of SoulBoundTokens(SBT) were presented.
+
+## Motivation
+
+We found a potential problem, the lack of credibility and faithfulness in online certifications, and imagined a solution, inspired by the everyday more common articles and mentions of Soulbound Tokens. We set to combine the best characteristics of each standardized EIP with the sole objective of elaborating a solid and useful token standard.
+
+### Characteristics
+* The NFT will be nontransferable after the initial transfer(SoulBoundToken-SBT).
+* Is backwards compatible with EIP-1155.
+* Double Signature.
+* Multi-Token.
+* Semi-Fungible standard.
+
+### Applications
+* Academy
+* Certifications
+* Smart Contract certifications(Code auditories)
+* POAP
+* And more
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+**Smart contracts implementing the SBTERC1155DS standard MUST implement all of the functions in the SBTERC1155DS interface.**
+
+**Smart contracts implementing the SBTERC1155DS standard MUST implement the ERC-165 supportsInterface function and and MUST return the constant value true if 0x812b9fae is passed through the interfaceID argument.They also MUST implement the EIP-1155 Interface and MUST return the constant value true if 0xd9b67a26 is passed through the interfaceID argument. Furthermore, they MUST implement the EIP-1155 Metadata interface, and MUST return the constant value true if 0x0e89341c is passed through the interfaceID argument.**
```

Comment:
```suggestion
**Smart contracts implementing this EIP MUST implement the [EIP-165](./eip-165.md) `supportsInterface` function and and MUST return the constant value true if `0x812b9fae` is passed through the `interfaceID` argument. They also MUST implement the EIP-1155 Interface and MUST return the constant value `true` if `0xd9b67a26` is passed through the `interfaceID` argument. Furthermore, they MUST implement the EIP-1155 Metadata interface, and MUST return the constant value true if `0x0e89341c` is passed through the `interfaceID` argument.**
```
Required: Self-reference as "this EIP"
Required: Grammar
Required: Reference as `EIP-<number>`
Required: Link on the first mention of an EIP
Recommended: Formatting